controllers/glanceapi_controller.go

```diff
@@ -639,16 +668,30 @@ func (r *GlanceAPIReconciler) generateServiceConfigMaps(
 
 	keystoneAPI, err := keystone.GetKeystoneAPI(ctx, h, instance.Namespace, map[string]string{})
 	if err != nil {
-		return err
+		errs = append(errs, err)
```

Comment:
This and similar checks here should remain `return err`.  We can't continue and assume we have the associated `keystoneAPI` object, for instance, if there was in fact an error.  

Did you see the `k8serrors.NewAggregate` pattern elsewhere (other operators in other projects)?  I have not seen that approach taken in operators before.

Reply:
I see your point: I created this error within the `generateServiceConfigMap` function to make sure any issue related to any component in the `ConfigMap` generation is returned and printed (as an error) within the operator log, otherwise I didn't see anything reporting error in the operator.
This can be useful during the development, but also to understand where is the issue.
We used this kind of pattern in [1], but I understand we'd like to rework the way we collect and print error logs.

[1] https://github.com/openshift/csi-driver-manila-operator/